fe2-android/app/src/main/java/com/github/dedis/popstellar/repository/remote/LAONetworkManager.java

```diff
@@ -106,10 +106,12 @@ private void processIncomingMessages() {
   public Completable catchup(Channel channel) {
     Log.d(TAG, "sending a catchup to the channel " + channel);
     Catchup catchup = new Catchup(channel, requestCounter.incrementAndGet());
+
     return request(catchup)
         .map(ResultMessages.class::cast)
         .map(ResultMessages::getMessages)
         .doOnSuccess(messages -> Log.d(TAG, "Catchup on " + channel + " retrieved : " + messages))
+        .doOnError(msg -> Log.d(TAG, "Error in catchup is " + msg))
```

Comment:
Isn't `doOnError()` supposed to provide a `Throwable` ?
And by design, we chose to keep the error handling to the caller. But this is a questionable choice.

Reply:
My mistake, I misunderstood the purpose of onError